src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs

```diff
@@ -1,4 +1,5 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
White space at the end was cleaned up automatically - I think it's ok to keep the change.